src/styles/blocks/header.scss

```diff
@@ -0,0 +1,173 @@
+  &__info-block {
+    padding: 0 20px 48px;
+    grid-column: span 2;
+
+    @include for-tablet {
+      padding: 0 0 147px;
+      grid-column: span 6;
+    }
+
+    @include for-desktop {
+      padding: 120px 0 0;
+      grid-column: span 6;
+    }
+
+    @include for-large {
+      padding: 120px 0 0;
+      grid-column: span 6;
+    }
+  }
+
+  &__navigation {
+    padding: 20px 20px 0;
+    margin-bottom: 48px;
+    grid-column: span 2;
+
+    @include for-tablet {
+      padding: 20px 0 0;
+      margin-bottom: 56px;
+      grid-column: span 6;
+    }
+
+    @include for-desktop {
+      padding: 48px 0 0;
+      margin-bottom: 44px;
+      grid-column: span 12;
+    }
+
+    @include for-large {
+      padding: 48px 0 0;
+      margin-bottom: 44px;
+      grid-column: span 12;
+    }
+  }
+
+  &__title {
+    display: block;
+    margin-bottom: 32px;
+    font-family: $font-primary;
+    font-size: 52px;
+    font-weight: 600;
+    line-height: 62px;
+    letter-spacing: -2px;
+    text-align: left;
+    color: $text-color;
+
+    @include for-tablet {
+      margin-bottom: 24px;
+      font-size: 64px;
+      line-height: 68px;
+    }
+
+    @include for-desktop {
+      margin-bottom: 32px;
+      font-size: 64px;
+      line-height: 68px;
+    }
+
+    @include for-large {
+      width: 490px;
+      margin-bottom: 32px;
+      font-size: 64px;
+      line-height: 68px;
+    }
+  }
+
+  &__paragraph {
+    display: inline-block;
+    margin-bottom: 72px;
+    font-family: $font-secondary;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 27px;
+    text-align: left;
+    color: $text-color;
+
+    @include for-tablet {
+      width: 362px;
+      margin-bottom: 56px;
+      grid-column: span 4;
+    }
+
+    @include for-desktop {
+      width: 362px;
+      margin-bottom: 83px;
+    }
+
+    @include for-large {
+      width: 362px;
+      margin-bottom: 83px;
+    }
+  }
+
+  &__button {
+    grid-column: span 2;
+
+    @include for-tablet {
+      grid-column: span 3;
+      width: 264px;
+    }
+
+    @include for-desktop {
+      grid-column: span 3;
+      width: 264px;
+    }
+
+    @include for-large {
+      grid-column: span 3;
+      width: 264px;
+    }
+  }
+
+  &__slider-block {
+    width: 100%;
+    height: 390px;
+
+    @include for-tablet {
+      grid-column: span 6;
+    }
+
+    @include for-desktop {
+      height: 680px;
+      grid-column: span 6;
+    }
+
+    @include for-large {
+      height: 680px;
+      grid-column: 8 / 13;
+    }
+  }
+
+  &__slider-image {
+    position: absolute;
+    width: 100%;
+    left: 0;
+    border-radius: 30px;
+    height: 390px;
+    object-fit: cover;
+
+    @include for-desktop {
+      width: calc(50% - (20px) / 2);
+      left: auto;
+      height: 680px;
+      border-radius: 30px 0;
+    }
+
+    @include for-large {
+      width: calc(50% - ((30px) / 2) - (30px + 68px));
+      left: auto;
+      height: 680px;
+      border-radius: 30px 0;
+    }
+  }
```

Comment:
A border radius should only be applied to the top-left corner of the image for screen widths between 1024px and 1600px.
![image](https://github.com/user-attachments/assets/9dea26fa-666c-4405-83d7-798a999afbe8)